bin/truffleruby

```diff
@@ -120,7 +138,28 @@ if [ -n "$CP" ]; then
 fi
 
 declare -a full_command
-full_command=("$JAVACMD" $JAVA_OPTS "-Xbootclasspath/a:$boot_cp_jar" "${java_args[@]}" org.truffleruby.Main "-Xhome=$root" "-Xlauncher=$root/bin/truffleruby" "${ruby_args[@]}")
+if [ $on_graalvm = false ]; then
+    full_command=(
+        "$JAVACMD"
+        $JAVA_OPTS
+        "-Xbootclasspath/a:$boot_cp_jar"
+        "${java_args[@]}"
+        org.truffleruby.Main
+        "-Xhome=$root"
+        "-Xlauncher=$root/bin/truffleruby"
+        "${ruby_args[@]}"
```

Comment:
The `isn't it?` was meant as "that's what I believe" and also asking to check by yourself so you get more confidence about it.
Look in `graalvm-0.20/language/ruby`, that's the home, where there is `lib`, `doc` and soon `bin`.